src/prototype_rust_sdk/workflow_context.rs

```diff
@@ -335,38 +390,76 @@ impl ChildWorkflowOptions {
 #[derive(Default, Debug)]
 pub struct ChildWorkflow {
     opts: ChildWorkflowOptions,
-    sequence_number: Option<u32>,
+}
+
+pub struct PendingChildWorkflow {
+    pub(crate) child_wf_cmd_seq_num: u32,
+    pub status: ChildWorkflowStartStatus,
+}
+
+impl PendingChildWorkflow {
+    /// Returns `None` if the child did not start successfully. The returned [StartedChildWorkflow]
+    /// can be used to wait on, signal, or cancel the child workflow.
+    pub fn as_started(&self) -> Option<StartedChildWorkflow> {
+        match &self.status {
+            ChildWorkflowStartStatus::Succeeded(s) => Some(StartedChildWorkflow {
+                child_wf_cmd_seq_num: self.child_wf_cmd_seq_num,
```

Comment:
Why does the WF code need to be aware of sequence numbers?

Reply:
The WF code itself isn't, this is inside the Rust "sdk"